internal/executor/instance/volume.go

```diff
@@ -20,9 +20,12 @@ var (
 	ErrVolumeCleanupFailed  = errors.New("failed to clean up working volume")
 )
 
+// AgentImage is the image we'll use to create a working volume.
+var AgentImage = "gcr.io/cirrus-ci-community/cirrus-ci-agent:v" + AgentVersion
```

Comment:
why `var`? can probably out in const after `AgentVersion`.

Reply:
Indeed, fixed in https://github.com/cirruslabs/cirrus-cli/pull/139/commits/5ddd4af7dd388e6098b0e3e541016a1d8e0f81b4.